champss/folding/multiday_search/fold_multiday.py

```diff
@@ -32,8 +32,8 @@ def find_all_dates_with_data(ra, dec, basepath, nday=0):
 
         date = dt.datetime(year, month, day)
 
-        datelow = dt.datetime(2024, 1, 31)
-        datehigh = dt.datetime(2024, 12, 31)
+        datelow = dt.datetime(2024, 1, 31)  # Are those ranges needed?
```

Comment:
They shouldn't be hardcoded, but probably good to have as an option.  The phase from ft/dt goes with t^2, so if you have a giant time range or big gap, the search space blows up quickly.  I think in this case, it was to avoid fold e.g. one extra single observation from ~6 months prior, and searchng over that